administration/general/network.rst

```diff
@@ -11,34 +11,40 @@ Hostname and domain settings.
+The grid displays only network interfaces configured through the |webui|. It is normal to see this panel empty if your default interface was configured by DHCP or static IP address during installation. The installer does not setup the network using |omv|, it just configures :file:`/etc/network/interfaces`.
+
+The dashboard contains a network widget that displays current status of interfaces.
 
 
 Ethernet
 ^^^^^^^^
 
-Just select DHCP or static. |omv| is a server so the recommended setting is to have static IP address. If you have a proper network infrastructure (separate router and switch ). In a reboot if the router fails to boot, you can still access the |webui| through the switch bridge. If the switch also fails you can use a direct ethernet connection as you already know the IP address assigned to the server NIC.
+Just select DHCP or static. |omv| is a server so the recommended setting is to have static IP address, if you have a proper network infrastructure (separate router and switch ). In a reboot, if the router fails to boot you can still access the |webui| through the switch bridge. If the switch also fails you can use a direct Ethernet connection with the static IP address assigned to the server NIC.
+
+When using static configuration, be aware that the configuration window does not expand completely. Scroll down for the IPv6 fields and DNS fields. 
 
-When using static configuration be aware that the configuration window does not expand completely, scrolling down are the IPv6 fields and DNS fields. The DNS setting is essential for fetching updates, do not leave empty. A common value is to use the same IP address as the gateway, if unsure just use google DNS ``8.8.8.8``.
+Do not leave DNS setting empty; it is essential for fetching updates. A common value is to use the same IP address as the gateway. If unsure, just use google DNS ``8.8.8.8``.
 
 Wake on LAN (WOL)
 	This enables WOL in the kernel driver, make sure the NICs supports this and the feature is enabled in BIOS.
 
 Wireless
 ^^^^^^^^
 
-Support for wireless network was added in |omv| 3.0. The configuration window displays the same IP configuration fields as ethernet, plus the relevant wireless values: SSID (the wireless network name) and the password. Please be aware wireless should not be used in a production server. This feature is intended for extreme cases. Whenever is possible please always use ethernet for a NAS server.
+Support for wireless network was added in |omv| 3.0. The configuration window displays the same IP configuration fields as Ethernet, plus the relevant wireless values: SSID (the wireless network name) and the password.
+
+Whenever possible,  use Ethernet for a NAS server. Wireless should not be used in a production server; this feature is intended for extreme cases only. 
 
 
 Vlan
 ^^^^
 
-If your network supports vlan, just add the parent interface and the VLAN id.
+If your network supports Vlan, just add the parent interface and the VLAN id.
```

Comment:
Why not writing all in upper case? At the end of the line VLAN is in upper case, too.

```suggestion
If your network supports VLAN, just add the parent interface and the VLAN id.
```